### PR TITLE
Fix utils include not being installed in header only

### PR DESCRIPTION
**Pull request by depaulmillz**
The util include files are not being installed currently when installing with header only set to on. This makes it difficult to use cutlass if it is installed as a header only library. This is a potential fix for that. This issue is described in #387.

### CODE REVIEW

**Comment by d-k-b** [comment]
@depaulmillz, sorry for the delay, I will try to review this soon.

**Review by d-k-b — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -67,21 +67,23 @@ set(CUTLASS_ENABLE_HEADERS_ONLY OFF CACHE BOOL "Enable only the header library")
...
 set(CUTLASS_ENABLE_EXAMPLES ${CUTLASS_ENABLE_EXAMPLES_INIT} CACHE BOOL "Enable CUTLASS Examples")
 set(CUTLASS_ENABLE_TOOLS ${CUTLASS_ENABLE_TOOLS_INIT} CACHE BOOL "Enable CUTLASS Tools")
-set(CUTLASS_ENABLE_LIBRARY ${CUTLASS_ENABLE_TOOLS} CACHE BOOL "Enable CUTLASS Library")
-set(CUTLASS_ENABLE_PROFILER ${CUTLASS_ENABLE_TOOLS} CACHE BOOL "Enable CUTLASS Profiler")
+set(CUTLASS_ENABLE_LIBRARY ${CUTLASS_ENABLE_LIBRARY_INIT} CACHE BOOL "Enable CUTLASS Library")
+set(CUTLASS_ENABLE_PROFILER ${CUTLASS_ENABLE_LIBRARY} CACHE BOOL "Enable CUTLASS Profiler")
 
 if(${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
-  set(CUTLASS_ENABLE_TESTS_INIT ${CUTLASS_ENABLE_TOOLS_INIT})
+  set(CUTLASS_ENABLE_TESTS_INIT ${CUTLASS_ENABLE_LIBRARY_INIT})
```
Comment (d-k-b) [inline CMakeLists.txt:86]: This should use `${CUTLASS_ENABLE_LIBRARY}` to capture user override.

**Review by d-k-b — CHANGES_REQUESTED** [review]
One minor fix needed.

File: CMakeLists.txt (hunk 1 above)
Comment (depaulmillz) [reply CMakeLists.txt]: Done.

**Review by d-k-b — APPROVED** [review]

**Pull request merged**